.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Auto detect text files and perform LF normalization
```

Comment:
is this new file necessary?

Reply:
You can probably remove it, I was having problems with the file endings and was hoping that would fix it.

Reply:
Did it fix your problems? i think we might keep the part about file endings if it helps windows users

Reply:
I did a lot of things, not least complete reinstall and futzing with my git settings, but it *might* have? Not entirely sure at this point (I had a cold and didn't think to do my usual troubleshooting documentation).

Reply:
okay, can you keep just the first line here and remove the one for visual studio and everything below it?

Reply:
For some reason, this is causing a bunch of sprite PNG files to get messed with when I check out this branch.

Reply:
In any case, I don't think it belongs in this PR. If it helps for your local dev environment, maybe leave it there but don't push it to the upstream branch?

Reply:
We have another or open with a similar change but I think the problem here is that the * targets all files while we should only target text files